src/components/common/NameRequestInfo.vue

```diff
@@ -1,14 +1,9 @@
 <template>
-  <div id="name-request-summary">
-    <ConfirmDialog
-      ref="confirmTranslationRemovalDialog"
-      attach="#name-request-summary"
-    />
```

Comment:
I took all the name translation stuff out of here and moved it to a new component, where I can reuse it independently from THIS component..